packages/frontend/src/components/footer.css

```diff
@@ -1,5 +1,5 @@
 .footer {
     /* background-image: url("../footer_bg.svg"); */
     /* background-color: rgb(174, 170, 170); */
-    padding: 4.5em 5em;
+    padding: 4.5em 3em;
```

Comment:
There's a mixture between `px` and `em` in the css files. We should use one everywhere. I'd recommend using `rem` as the unit, because this will not be affecting by font changes in elements.